### ISSUE TITLE
crash on exit

### ISSUE THREAD

**Issue by aremazeilles** [aremazeilles/beat_routine#23]
Omitting on purpose an input parameter:

```term
./run_protocol1 tests/data/protocol1/input/jointAngles.csv tests/data/protocol1/input/platformData.csv tests/data/protocol1/input/emg.csv         

warning: function beat/AdditionalFunctions/corrcoef.m shadows a core library function
warning: called from
    ./run_protocol1 at line 5 column 1
ERROR: Provided 3 parameters, but expecting 4
Usage:
	 run_protocol1 jointAngle.csv platformData.csv emg.csv out_folder
terminate called after throwing an instance of 'octave::exit_exception'
panic: Aborted -- stopping myself...
attempting to save variables to 'octave-workspace'...
save to 'octave-workspace' complete
octave exited with signal 6
```

Have a look at [here](https://stackoverflow.com/questions/57940972/octave-not-returning-exit-code). It seems to be a bug within octave 4.2.
Locally I have octave 4.4.2. Should check the version in the docker image

**Comment by aremazeilles** [comment]
Trial with the docker image gives:
docker run --rm -v $PWD/tests/datat/protocol1/input:/in -v $PWD/out_tests:/out beat_routine ./run_protocol1 /in/jointAngles.csv /in/platformData.csv /in/emg.csv     
octave: X11 DISPLAY environment variable not set
octave: disabling GUI features
warning: function beat/AdditionalFunctions/corrcoef.m shadows a core library function
warning: called from
    ./run_protocol1 at line 5 column 1
ERROR: Provided 3 parameters, but expecting 4
Usage:
	 run_protocol1 jointAngle.csv platformData.csv emg.csv out_folder
terminate called after throwing an instance of 'octave::exit_exception'
panic: Aborted -- stopping myself...
attempting to save variables to 'octave-workspace'...
save to 'octave-workspace' complete
panic: attempted clean up failed -- aborting...

**Comment by aremazeilles** [comment]
To install  newer version, have a look at [this link](http://ubuntuhandbook.org/index.php/2018/07/install-latest-gnu-octave-4-4-ubuntu-18-04/)

**Comment by juritaborri** [comment]
@aremazeilles Have I to perform some actions for this issue?

**Comment by aremazeilles** [comment]
No, but please leave it open. This is something I would like to fix at a moment.
